app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/restaurant/details/RestaurantDetailsScreen.kt

```diff
@@ -0,0 +1,140 @@
+package co.co.nimblehq.showcases.poly.place.ui.screens.restaurant.details
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.hilt.navigation.compose.hiltViewModel
+import co.co.nimblehq.showcases.poly.place.R
+import co.co.nimblehq.showcases.poly.place.ui.AppDestination
+import co.co.nimblehq.showcases.poly.place.ui.common.StartDrawableText
+import co.co.nimblehq.showcases.poly.place.ui.screens.restaurant.uimodel.Restaurant
+import co.co.nimblehq.showcases.poly.place.ui.theme.*
+import co.co.nimblehq.showcases.poly.place.ui.theme.AppTheme.dimensions
+
+@Composable
+fun RestaurantDetailsScreen(
+    viewModel: RestaurantDetailsViewModel = hiltViewModel(),
+    navigator: (destination: AppDestination) -> Unit,
+    restaurant: Restaurant?
+) {
+
+    restaurant?.let {
+        RestaurantDetailsContent(
+            restaurant = it,
+            onBackButtonClicked = { navigator(AppDestination.Up) }
+        )
+    }
+}
+
+@Composable
+private fun RestaurantDetailsContent(restaurant: Restaurant, onBackButtonClicked: () -> Unit) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(ViolentViolet)
+            .padding(horizontal = dimensions.spacingNormal)
+            .padding(top = dimensions.spacingHuge)
+    ) {
+        RestaurantDetailsToolbar(
+            title = restaurant.name,
+            onBackButtonClicked = onBackButtonClicked
+        )
+        RestaurantDetails(restaurant = restaurant)
+    }
+}
+
+@Composable
+private fun RestaurantDetailsToolbar(
+    title: String,
+    modifier: Modifier = Modifier,
+    onBackButtonClicked: () -> Unit
+) {
+    Column(modifier) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
```

Comment:
```suggestion
    Row(
       modifier = modifier,
       verticalAlignment = Alignment.CenterVertically
    ) {
```

I think we can remove this `Column` since there is just 1 `Row` in its children 🤔 
Please feel free to correct me if I was wrong 😄

Reply:
Nice catch! @thiennguyen0196